packages/playground/src/utils/nodeSelector.ts

```diff
@@ -235,10 +234,13 @@ export async function validateRentContract(
         id: node.rentContractId,
       });
       if (contractInfo.state.gracePeriod) {
-        createCustomToast(
-          `You can't deploy on node ${node.nodeId}, its rent contract is in grace period.`,
-          ToastType.danger,
-        );
+        const err = `You can't deploy on node ${node.nodeId}, its rent contract is in grace period.`;
+        await new Promise((_, reject) => {
+          setTimeout(() => {
+            reject(Error(err));
+          }, 2000);
+        });
```

Comment:
i don't think we need to add timeout here, not the auto one there is no toast overlap what do you think?

Reply:
totally agree, removed

Reply:
I see you removed timeout in the automatic on to, can you explain why I think ti is a better UX even if we set it to only one sec